src/components/flash/flash.js

```diff
@@ -223,19 +233,23 @@ class Flash extends React.Component {
    * @param {Object} ev
    * @return(Void)
    */
-  toggleDialog = (key, ev) => {
-    if (ev) { ev.preventDefault(); }
-
-    let state = this.state.dialogs[key];
-    // open/close the dialog
-    this.setState({ dialogs: { [key]: !state }});
-
-    // start/stop the timer if the dialog opens or closes
-    if (state) {
-      this.startTimeout();
-    } else {
-      this.stopTimeout();
-    }
+  toggleDialog = (key) => {
```

Comment:
Yeah, im happy. I don't know if for consistency we want to do it once way.

@Sage/carbon may have other opinions. I would keep going this way for now though